test/DebugInfo/ImportsStdlib.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend -emit-ir -parse-stdlib -module-name NotTheStdlib %s -I %t -g -o - > %t.ll
+// RUN: %FileCheck %s < %t.ll
+// RUN: %FileCheck -check-prefix=NEGATIVE %s < %t.ll
+
+// RUN: %target-swift-frontend -c -parse-stdlib -module-name NotTheStdlib %s -I %t -g -o %t.o
+// RUN: %llvm-dwarfdump -a %t.o > %t.dump
```

Comment:
Just checking the IR would have been sufficient — we already check that a DIImportedEntity(DIModule) gets lowered correctly elsewhere, but this is fine, too.

Reply:
Ah, right. I copied from the other test and just updated everything.